test/Transforms/Passes/tuple-to-struct.mlir

```diff
@@ -0,0 +1,99 @@
+// RUN: p4mlir-opt --p4hir-tuple-to-struct %s | FileCheck %s
```

Comment:
it is better to use `--split-input-file` here to ensure that things are really independent